python/rascaline/transformer.py

```diff
@@ -0,0 +1,296 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+from equistore import Labels, TensorBlock, TensorMap
+
+
+class Transformer:
```

Comment:
how about something like this? Then explaining in the docstring why it is useful.

```suggestion
class PropertiesSelector:
```

Reply:
Yep, sounds great!